src/framework/theme/components/user/user.component.ts

```diff
@@ -199,6 +199,10 @@ export class NbUserComponent {
   showInitialsValue: boolean = true;
   isMenuShown: boolean = false;
 
+  get onlyPictureMode(): boolean {
+    return this.showNameValue || this.showTitleValue;
+  }
```

Comment:
This getter lies me 😸, `onlyPictureMode` has to be true while we shouldn't `showNameValue` and `showTitleValue`. But now it does the inverse thing - it's true if we have to show either name or title or even both.
I think you have to add negation in the getter. Otherwise, it'll keep on lying.

Reply:
You're right. My bad.